vscode-client/package.json

```diff
@@ -55,6 +55,11 @@
                     "default": true,
                     "description": "Auto-completes methods and classes with tabstops for each parameter."
                 },
+                "pyls.plugins.jedi_completion.fuzzy": {
+                    "type": "boolean",
+                    "default": true,
```

Comment:
Please change this to `false`, because the setting is false by default, right?

Reply:
@ccordoba12 see affc871